source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -0,0 +1,332 @@
+1100
+~~~~
+
+## Prerequisites
+Setting the MTU via libvirt will work only on hosts with RHEL 7.4 or later.
+For hosts with RHEL 7.4 or later libvirt's XML is produced by engine, which will
+contain the definition. On hosts running older software, vdsm will create
+libvirt's XML without the MTU definition.
+Setting the MTU via libvirt works only on VMs with machine type rhel7.4.0 or
+later, so setting the MTU via libvirt will work transparent to the user if the
+default machine type in oVirt is rhel7.4.0 or later.
+
+Setting the MTU by DHCP individally per network will require an updated
+version of [OpenStack Java SDK][11] used by oVirt-engine.
+
+## Limitations
+
+* MTU is applied during the start of the VM, but not during an update of the
+  network.
+
+* Setting the MTU by DHCP requires that the relevant interface is configured
+  via DHCP by the guest.
+
+* Setting the MTU via libvirt requires that the machine type of the VM is
+  rhel7.4.0 or later, which requires that the hosts are running RHEL 7.4 or
+  later.
+
+* Setting the MTU via libvirt requires that vNIC interface is of model type
+  'virtio' and the VirtIO driver of the guest supports MTU negation.
+  This is given for RHEL 7.4 or later and VirtIO-Win version 1.9.1-0 (netkvm
+  build 139) or later, which is already given for the already released
+  oVirt-toolsSetup 4.2-1.
+
+## Benefit to oVirt
+
+If oVirt assist to configure the MTU inside the guest VM helps to get the VM
+working as intended by the user.
+
+If the MTU inside the guest VM is set too large, too large Ethernet frames sent
+by the VM are dropped without any hint to the user.
+If the MTU inside the guest VM is set too small, the performance of the vNIC
+might be below the users expectation.
+
+## Entity Description
+
+The MTU property has to be enabled for creating external networks in webadmin,
+the related AddNetworkOnProviderCommand, and during import and synchronization.
+Similar to network name, the MTU of an external network cannot be updated by
+engine.
+
+### Default Value for MTU
+If no MTU is defined for a network, [DHCP](#set-the-mtu-by-dhcp) falls back on
+a default value configured in [oVirt OVN Provider][1].
+
+Inside oVirt Engine every logical network has a MTU. If the user does not
+specify a MTU explicitly, the MTU has the value `default`.
+During [defining the vNIC by libvirt](#set-the-mtu-by-libvirt) the value
+`default` is mapped to the default MTU for host networks, usually 1500 bytes,
+or a new configuration value to configure the MTU of external networks, usually
+1442 bytes.
+If the value of MTU is `default`, the MTU is not send to the external provider,
+because [OpenStack's mtu property][16] is optional and may confuse the provider.
+
+## User Experience
+
+The already available MTU property of logical network is enabled for external
+networks.
+
+![edit an external network Engine dialog](images/features/network/managed-mtu-for-vms_edit_mtu.png)
```

Comment:
the value shown there is imprecise due to tunneling overhead, but we did not find a nicer solution.

Reply:
Done